index.js

```diff
@@ -186,6 +186,8 @@ function createExtension(name, { overridePage, devtools, language }) {
     watch:
       'webpack --mode=development --watch --config config/webpack.config.js',
     build: 'webpack --mode=production --config config/webpack.config.js',
+    pack: 'node pack.js',
+    repack: 'npm run build && npm run pack',
```

Comment:
Let's mention these commands in README file after `npm run build` command - https://github.com/dutiyesh/chrome-extension-cli/blob/master/README.md?plain=1#L106.

Also in same README file there are few places where we have mentioned to zip the `build` folder so we can replace it and instead mention to run `npm run pack` command.
- Reference 1: https://github.com/dutiyesh/chrome-extension-cli/blob/master/README.md?plain=1#L109
- Reference 2: https://github.com/dutiyesh/chrome-extension-cli/blob/master/README.md?plain=1#L30